### PR TITLE
Update Russian Translations

### PR DESCRIPTION
**Pull request by oleg-nenashev**
# Description

The change integrates #2657 into the core

- [x] Squash the original commits
- [x] Resolve merge conflicts
- [x] Ensure that the introduced hyperlinks use redirectors (like in #2756)

### Changelog entries

Proposed changelog entries:

* Update Russian Translations
* ...

### Submitter checklist

- [x] JIRA issue is well described
- [x] Link to JIRA ticket in description, if appropriate
- [x] Appropriate autotests or explanation to why this change has no tests
- [x] For new API and extension points: Link to the reference implementation in open-source (or example in Javadoc)

### Desired reviewers

@mention

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
CC @daniel-beck and @bigscoop

**Comment by oleg-nenashev** [comment]
The text itself has been reviewed and approved by me in #2657 .
I rather seek an approval for the merge after the minor changes

**Comment by daniel-beck** [comment]
Assuming this is just about commits attributed to Oleg, :+1:

File: core/src/main/resources/lib/hudson/project/config-upstream-pseudo-trigger_ru.properties [hunk 1]
```diff
@@ -20,7 +20,3 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
```
Comment (daniel-beck) [inline core/src/main/resources/lib/hudson/project/config-upstream-pseudo-trigger_ru.properties]: @oleg-nenashev *This* is why I now have a merge conflict in #2777? :cry:

Comment (oleg-nenashev) [inline core/src/main/resources/lib/hudson/project/config-upstream-pseudo-trigger_ru.properties]: @daniel-beck Why would you edit the Russian translation there? :)

**Pull request merged**